org.nicolasbock.ebuildtester.yaml

```diff
@@ -0,0 +1,21 @@
+app-id: org.nicolasbock.ebuildtester
```

Comment:
also `nicolasbock.org` doesn't resolve, so I question the validity of this app-id

Reply:
Can I use `io.readthedocs.ebuildtester` instead? I don't own a specific domain for this project.

Reply:
https://docs.flatpak.org/en/latest/conventions.html#application-ids

This would suggest to use github since that's where the code is.